confdgnmi/src/confd_gnmi_client.py

```diff
@@ -200,7 +212,8 @@ def parse_args(args):
                         type=int,
                         help="Number of read requests for STREAM subscription (default 4)",
                         default=4)
-    (opt, args) = parser.parse_known_args(args=args)
+    parser.add_argument("--encoding", choices=["BYTES", "JSON", "JSON_IETF"], default="JSON_IETF")
+    opt = parser.parse_args(args=args)
     log.debug("opt=%s", opt)
     return opt
 
```

Comment:
I guess the parameterized support of host, port, username, and password are being taken care of by Michal?

Reply:
Yes, they are, as a separate effort.